### PR TITLE
Hallp

### PR DESCRIPTION
**Pull request by Samaamro20**
- hall page BackEnd 
- hall page simple design
- Slider Dose not work 
- Need API to Map location 
- Need Calender

### CODE REVIEW

**Comment by Samaamro20** [comment]
- slider Dose not work

File: src/backEnd/controllers/hall.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+
+const { serviceDetails, media, user } = require('../../database/models');
+
+exports.get = (req, res) => {
+	const { params: { id } } = req;
+	// ////
```
Comment (shahenazmonia) [inline src/backEnd/controllers/hall.js]: delete commented lines

File: src/backEnd/controllers/hall.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	// const details= result[0].service_detail.dataValues
+	// const images=    result.map((value)=>{
+	//     return value.image_url
+	//     })
+	//     const filteredResult= {images,details}
+	//         res.send(filteredResult);
+	// ///
+	media.findAll({
+		where: { serviceDetailId: id },
```
Comment (shahenazmonia) [inline src/backEnd/controllers/hall.js]: fix indentation here and in other files

File: src/backEnd/controllers/hall.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+		const details = result[0].service_detail.dataValues;
+		const images = result.map(value => value.image_url);
+		const filteredResult = [{ images, details }];
+		// res.send(filteredResult);
+		 res.render('hall', { filteredResult });
+	})
+		.catch(() => res.status(500).json({ err: 'error in query' }));
+// serviceDetails.findAll({ where : {id : id },
+// attributes: ['name', 'image_url','location','description','price','map_loc'],
+// raw: true })
```
Comment (shahenazmonia) [inline src/backEnd/controllers/hall.js]: Remove commented lines

File: src/backEnd/controllers/hall.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+			 }],
+		  }],
+	}).then((result) => {
+		const details = result[0].service_detail.dataValues;
+		const images = result.map(value => value.image_url);
+		const filteredResult = [{ images, details }];
+		// res.send(filteredResult);
+		 res.render('hall', { filteredResult });
+	})
+		.catch(() => res.status(500).json({ err: 'error in query' }));
```
Comment (shahenazmonia) [inline src/backEnd/controllers/hall.js]: Handle the error in another way, either render it to front-end or define server-error-function use ```next()```

**Pull request merged**